Cargo.toml

```diff
@@ -95,6 +93,6 @@ name = "write_netex_france"
 path = "tests/write_netex_france.rs"
 required-features = ["proj"]
 
-[dev-dependencies]
+[dev_dependencies]
```

Comment:
Hmm, official documentation mentioned with a dash `-`.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
I know, I didn't change that, weird